torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -477,6 +478,18 @@ struct CodeImpl {
   // map from unique of nodes to register in register table
   std::unordered_map<Value*, int> value_to_reg_;
 
+  // map from operator name to specified arguments
+  // Example: for a schema of aten::foo.str
+  // aten::foo.str(arg0: str="default", arg1: int=0,
+  //               arg2: bool=False, arg3: float=0.0)
+  // If the usages in a graph is:
+  //    aten::foo("somestr", arg1=0, arg2=False, arg3=0.0)
+  //    aten::foo("somestr", arg1=1, arg2=False, arg3=0.0)
+  // op_to_num_specified_args_["aten::foo.str"] = 2
+  // This is because for all usages, last two args are not used. (e.g. same
+  // value as default schema value)
```

Comment:
Number "2" here is symmetric, can be either specified or unspecified arg#. Suggest to change it to 3.
```suggestion
  //    aten::foo("somestr", arg1=1, arg2=True, arg3=0.0)
  // op_to_num_specified_args_["aten::foo.str"] = 3
  // This is because for all usages, at most first three args are specified. last one args are not used. (e.g. same
  // value as default schema value)
```

Reply:
done